micromouse/Motor.cpp

```diff
@@ -42,29 +123,51 @@ namespace Micromouse
 		{
 			digitalWrite(fwdPin, HIGH);
 			digitalWrite(bwdPin, LOW);
-			analogWrite(pwmPin, (int)(255 * speed));
+			setVoltage((int)(255 * speed), true);
 		}
 		else
 		{
 			digitalWrite(fwdPin, LOW);
 			digitalWrite(bwdPin, HIGH);
-			analogWrite(pwmPin, (int)(255 * (-speed)));
+			setVoltage((int)(255 * (-speed)), false);
 		}
-#endif
+	}
+
+
+	
+	void Motor::setVoltage(int voltage, bool fwd)
```

Comment:
![image](https://cloud.githubusercontent.com/assets/4276762/17644744/c8cf19a8-6143-11e6-9cf5-3647aeefb9d1.png)

So if I understand this correctly, the speed maps to voltage like depicted on the left

do you think a voltage mapping like on the right be better?